CHANGELOG.md

```diff
@@ -6,10 +6,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Deprecate FFI definition `PyCoro_Check` and `PyAsyncGen_Check` in favor of `PyCoro_CheckExact` and `PyAsyncGen_CheckExact` respectively, as these are the correct name in the Python headers. [#1341](https://github.com/PyO3/pyo3/pull/1341)
```

Comment:
👍 changelog message looks good, but I don't see the `#[deprecated]` attribute in `genobject.rs`?

```suggestion
- Deprecate FFI definitions `PyCoro_Check` and `PyAsyncGen_Check` in favor of `PyCoro_CheckExact` and `PyAsyncGen_CheckExact` respectively (the former are not part of the Python API). [#1341](https://github.com/PyO3/pyo3/pull/1341)
```

Reply:
Oops, I accidentally staged this. Removing.